app/gui/language/ast/impl/src/macros/skip_and_freeze.rs

```diff
@@ -0,0 +1,144 @@
+//! Helper functions and data structures for [`SKIP`] and [`FREEZE`] macros support.
+//!
+//! [`SKIP`] and [`FREEZE`] macros change the behavior of the expression they are prepending.
+//!
+//! When used together in a single expression, [`SKIP`] macro always takes the priority and is
+//! placed at the beginning. For example, `SKIP FREEZE foo`.
```

Comment:
```suggestion
//! Helper functions and data structures for `SKIP` and `FREEZE` macros support.
//!
//! `SKIP` and `FREEZE` macros change the behavior of the expression they are prepending.
//!
//! When used together in a single expression, `SKIP` macro always takes the priority and is
//! placed at the beginning. For example, `SKIP FREEZE foo`.
```
Maybe I miss something, but SKIP and FREEZ are not identifiers visible in this module.

Reply:
There were multiple cases in the past when @wdanilo asked me to add these brackets even though they do not work as links. So I'm leaving them as they are, it's not a big deal.

Reply:
Hmm, and wasn't that a different case? When we had a normal comment (not doc-comment) but still refer to existing entities?

Reply:
Here are a few cases: https://github.com/enso-org/enso/pull/3520#discussion_r896674837